packages/api/models/providers.js

```diff
@@ -1,6 +1,6 @@
 'use strict';
 
-const Crypto = require('@cumulus/ingest/crypto').DefaultProvider;
```

Comment:
When you say naming this, which part do you mean?  I only renamed it from crypto because there's already a 'crypto' lib required and a crypto.js added to kes.  It seemed like a reasonable compromise to name it what I thought it did.  (but I can think wrongly)

Reply:
I mean the variable name. It's imported as `{ DefaultProvider }` everywhere else. I don't really have a preference for that name or this name, but we should choose one.

Reply:
gotcha, I can change that.

Reply:
519539f6e